doc/howto/cluster_manage_instance.md

```diff
@@ -37,3 +37,8 @@ For example, to move the instance `c1` to the cluster member `server1`, use the
     lxc start c1
 
 See {ref}`move-instances` for more information.
+
+To move an instance on a member of a cluster group, use the group name prefixed with `@` for the `--target` flag.
```

Comment:
```suggestion
To move an instance to a member of a cluster group, use the group name prefixed with `@` for the `--target` flag.
```
Just to avoid ambiguity. :)

Reply:
Thanks, applied.